exaudfclient/base/javacontainer/javacontainer_impl.cc

```diff
@@ -433,7 +433,7 @@ void JavaVMImpl::registerFunctions() {
 }
 
 void JavaVMImpl::setClasspath() {
-    m_exaJarPath = m_exaJavaPath + "/libexaudf.jar";
+    m_exaJarPath = m_exaJavaPath + "/exaudf_deploy.jar";
```

Comment:
According to the [Bazel docu](https://bazel.build/reference/be/java#java_binary_implicit_outputs) this needs to be `exaudf_deploy.jar` because otherwise the external dependencies (UDF API!!!) are missing. Not sure how this worked before: I guess this is also a breaking change due to the new Bazel version.